packages/website/pages/docs/guides/run-a-node/withdraw-ttko-balance.mdx

```diff
@@ -31,7 +29,7 @@ For example if you want to withdraw `69` TTKO, you would enter `6900000000`.
 <Callout type="warning">
 Make sure
 ```solidity
-amount<=withdrawTaikoToken(proverAddress)
+amount<=getTaikoTokenBalance(proverAddress)
 ```
```

Comment:
i don't think this should be a solidity snippet but rather plain english. suggestion:

```markdown
Make sure that you have enough balance of TTKO/TTKOe for the amount you are withdrawing, or the transaction will revert. You can check your balance with the `getTaikoTokenBalance()` function.
```

Reply:
I agree, could you make this a code suggestion?

Reply:
it won't let me because i can only suggest on the affected lines "can't delete the `Make sure`" part